tests/functional/adapter/incremental/test_incremental_tblproperties.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from dbt.tests import util
+from tests.functional.adapter.incremental import fixtures
+
+
+class TestIncrementalTblproperties:
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "merge_update_columns_sql.sql": fixtures.merge_update_columns_sql,
+            "schema.yml": fixtures.tblproperties_a,
+        }
+
+    # For now, only add/change, no deletes
+    def test_changing_tblproperties(self, project):
+        util.run_dbt(["run"])
+        util.write_file(fixtures.tblproperties_b, "models", "schema.yml")
+        util.run_dbt(["run"])
+        results = project.run_sql(
+            "show tblproperties {database}.{schema}.merge_update_columns_sql",
+            fetch="all",
+        )
+        results_dict = {}
```

Comment:
This test only asserts that the tblproperties that we set have been added/updated.  Databricks has been increasing the number of tblproperties that get set automatically, so not safe to assert that these are the only tblproperties on the table.